src/part07.rs

```diff
@@ -29,28 +29,32 @@ pub fn vec_min<T: Minimum>(v: &Vec<T>) -> Option<&T> {
 //@ C(++) or Java that can be `NULL`! However, thanks to `Option` being an `enum`, we cannot forget
 //@ to check the pointer for validity, avoiding the safety issues of C(++). <br/>
 //@ Also, if you are worried about wasting space, notice that Rust knows that `&T` can never be
-//@ `NULL`, and hence optimizes `Option<&T>` to be no larger than `&T`. The `None` case is represented
-//@ as `NULL`. This is another great example of a zero-cost abstraction: `Option<&T>` is exactly like
-//@ a pointer in C(++), if you look at what happens during execution - but it's much safer to use.
-
-// **Exercise 07.1**: For our `vec_min` to be usable with `BigInt`, you will have to provide an implementation of
-// `Minimum`. You should be able to pretty much copy the code you wrote for exercise 06.1. You should *not*
-// make any copies of `BigInt`!
+//@ `NULL`, and hence optimizes `Option<&T>` to be no larger than `&T`. The `None` case is
+//@ represented as `NULL`. This is another great example of a zero-cost abstraction: `Option<&T>`
+//@ is exactly like a pointer in C(++), if you look at what happens during execution - but it's
+//@ much safer to use.
+
+// **Exercise 07.1**: For our `vec_min` to be usable with `BigInt`, you will have to provide an
+// **implementation of `Minimum`. You should be able to pretty much copy the code you wrote for
```

Comment:
This and the next line should not start with `**`.

Reply:
I did go through the whole thing again once finished, but it looks like I didn't manage to catch all the wrong changes. :-(

Reply:
It's okay, that's what we do review for :)